tests/Fixer/PhpUnit/PhpUnitTestAnnotationFixerTest.php

```diff
@@ -931,14 +899,93 @@ class Test extends \PhpUnit\FrameWork\TestCase
     public function testWhyDoThis() {}
 }',
             ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideNoChangeFixCases
+     *
+     * @param string $input
+     * @param array  $config
+     */
+    public function testItDoesNotChange($input, array $config = [])
```

Comment:
please move it back to one test method / data provider

Reply:
Okay